src/components/form-field/single-combobox/single-combobox-input.tsx

```diff
@@ -2,28 +2,28 @@ import { Combobox as HeadlessCombobox } from "@headlessui/react";
 import React from "react";
 import { CaretDownIcon } from "../../../icons";
 
-export interface SingleComboboxInputProps {
+export interface SingleComboboxInputProps<TValue> {
     id: string;
-    displayValue: string;
+    displayValue?(item: TValue): string;
```

Comment:
This change should break the storybook example, no? 
I think we should modify/add something in the story to showcase this new behaviour

Reply:
Sorry, there was one commit missing. I added an example in Storybook for object values and removed the displayValue from the other examples.